pkg/iostreams/color.go

```diff
@@ -0,0 +1,86 @@
+package iostreams
+
+import "github.com/mgutz/ansi"
+
+var (
+	magenta = ansi.ColorFunc("magenta")
+	cyan    = ansi.ColorFunc("cyan")
+	red     = ansi.ColorFunc("red")
+	yellow  = ansi.ColorFunc("yellow")
+	blue    = ansi.ColorFunc("blue")
+	green   = ansi.ColorFunc("green")
+	gray    = ansi.ColorFunc("black+h")
+	bold    = ansi.ColorFunc("default+b")
+)
+
+func NewColorScheme(enabled bool) *ColorScheme {
```

Comment:
i like this approach to color! but please open a tech debt issue to port the whole codebase to it so we don't end up with competing color systems.